app/controllers/api/health.php

```diff
@@ -21,6 +21,20 @@
 use Utopia\Validator\Text;
 use Utopia\Validator\WhiteList;
 
+$checkStorageHealth = function (Device $device) {
```

Comment:
We try to avoid global functions as much as possible, also this one is called only once so I doubt we need this as a function vs just adding the code in the loop.